Android/src/org/droidplanner/android/proxy/mission/item/fragments/MissionDetailFragment.java

```diff
@@ -206,11 +208,69 @@ public void onSpinnerItemSelected(Spinner spinner, int position) {
         }
     };
 
+    private final Spinner.OnItemSelectedListener missionFrameListener = new Spinner.OnItemSelectedListener() {
+
+        @Override
+        public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+            frameSpinner = (Spinner) view.findViewById(R.id.frameSpinner);
+
+            if (frameSpinner != null && mSelectedProxies.isEmpty())
```

Comment:
@billbonney This has the effect that if `frameSpinner` is `null` the early return will be aborted. Is that the intended effect?